packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -146,6 +146,21 @@ class ScrollableBase extends Component {
 		 */
 		focusableScrollbar: PropTypes.bool,
 
+		/**
+		 * Specifies overscroll effects shows on which type of inputs.
+		 *
+		 * @type {String[]}
```

Comment:
The type might be Object or null, right?

Reply:
Oops, I missed to change it! Thanks.